opennms-webapp/src/main/webapp/about/index.jsp

```diff
@@ -131,10 +131,25 @@
     <th>Database Version:</th>
     <td><%=dbVersion%></td>
   </tr>
+<%
+    String globalMBStrategy, sinkMBStrategy, rpcMBStrategy;
+    globalMBStrategy = org.opennms.core.rpc.common.RpcStrategy.getRpcStrategy().getDescr();
+    if (globalMBStrategy != null) {
+        sinkMBStrategy = globalMBStrategy;
+        rpcMBStrategy = globalMBStrategy;
+    } else {
+        sinkMBStrategy = org.opennms.core.ipc.sink.common.SinkStrategy.getSinkStrategy().getDescr();
+        rpcMBStrategy = org.opennms.core.rpc.common.RpcStrategy.getRpcStrategy().getDescr();
+    }
+%>
   <tr>
     <th>Time-Series Strategy:</th>
     <td><%=TimeSeries.getTimeseriesStrategy().getDescr()%></td>
   </tr>
+  <tr>
+    <th>IPC Strategy:</th>
+    <td>Sink: <%=sinkMBStrategy%>; RPC: <%=rpcMBStrategy%></td>
```

Comment:
Depending on how it looks on the web page, we should choose between `Descr` and `name`.
I prefer using `name`